Packs/Stairwell/Integrations/Inception/Inception.py

```diff
@@ -0,0 +1,185 @@
+
+            # Create readable output if AV results exist
+            if "last_analysis_results" in response:
+                avResults = responseJson['data']['attributes']['last_analysis_results']
+                md += '### AV Scanning Results\n'
+                md += 'Engine Name|Result\n'
+                md += '---|---\n'
+                for indAv in avResults:
+                    engine_name = avResults[indAv]['engine_name']
+                    result = avResults[indAv]['result']
+                    md += f'{engine_name}|{result}\n'
+            results = CommandResults(
+                readable_output=md,
+                outputs_prefix='Inception.File_Details',
+                outputs=responseJson,
+            )
+            return(results)
+    except DemistoException as err:
+        # API will return 404 if the file is not found
+        if "404" in str(err):
+            results = CommandResults(
+                readable_output="File not found: " + file_hash
+            )
+            return(results)
+        else:
+            raise err
+
+
+def variant_discovery_command(client, file_hash):
+    try:
+        response = client.get_file_reputation(file_hash)
+        response = json.dumps(response)
+        if("similarity" in response):
+            response_json = json.loads(response)
+            md_string = tableToMarkdown("File Variants Discovered", response_json['variants'])
+            results = CommandResults(
+                outputs_prefix='Inception.Variants',
+                readable_output=md_string,
+                outputs=response_json,
+            )
+            return(results)
+        elif("variants" in response):
+            results = CommandResults(
+                readable_output="No variants discovered for: " + file_hash
+            )
+            return(results)
+    except DemistoException as err:
+        # API will return 500 if the file is not found
+        if "500" in str(err):
+            results = CommandResults(
+                readable_output="File not found: " + file_hash
+            )
+            return(results)
+        else:
+            raise err
+
+
+def main() -> None:  # pragma: no cover
+    params = demisto.params()
+    args = demisto.args()
+    command = demisto.command()
+
+    api_key = params.get('apikey', {}).get('password')
+
+    # Params enabled by XSOAR functionality
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    demisto.debug(f'Command being called is {command}')
+    try:
+        # Built-in command
+        if command == 'test-module':
+            # This is the call made when clicking the integration Test button.
+            client = Client(
+                base_url="https://reputation.app.stairwell.com/api/v3/files/",
+                verify=verify_certificate,
+                headers={"X-Apikey": api_key},
+                proxy=proxy)
```

Comment:
I have left these configured for each command given they have different "base_url"s and authentication headers. Given this, it requires unique Client configuration per command. This may improve as we enhance our APIs, and we'll consider collapsing when these become common.